vaadin-accordion-flow-demo/pom.xml

```diff
@@ -7,7 +7,7 @@
     <parent>
         <groupId>com.vaadin</groupId>
         <artifactId>vaadin-accordion-flow-root</artifactId>
-        <version>2.0-SNAPSHOT</version>
+        <version>2.1-SNAPSHOT</version>
```

Comment:
Nit: `radio-button` snapshot can be bumped to `2.1`

Reply:
Done.